### PR TITLE
Update bio-formats to 6.7.0

### PR DESCRIPTION
**Pull request by ian-keeneye**
This PR updates bio-formats to 6.7 and also suppresses the reading of the original metadata which we do not currently  use.

### CODE REVIEW

File: src/test/java/ai/keeneye/bioformats2n5/test/N5Test.java [hunk 1]
```diff
@@ -17,19 +17,24 @@
...
 import ai.keeneye.bioformats2n5.Downsampling;
 import loci.common.LogbackTools;
-import loci.common.services.ServiceFactory;
+
 import loci.formats.FormatTools;
 import loci.formats.in.FakeReader;
-import loci.formats.ome.OMEXMLMetadata;
-import loci.formats.services.OMEXMLService;
+
+// imports for originalMetadata tests
```
Comment (bench) [inline src/test/java/ai/keeneye/bioformats2n5/test/N5Test.java:32]: @ian-keeneye any value to those comments ?

File: src/test/java/ai/keeneye/bioformats2n5/test/N5Test.java [hunk 2]
```diff
@@ -491,10 +496,11 @@ public void testDownsampleEdgeEffectsUInt16() throws Exception {
     assertEquals(50, tile.get(75 * 4));
   }
 
-  /**
+  // restore this test if original metadata is required
+  /*
    * Test that original metadata is saved.
    */
-  @Test
+  /* @Test
```
Comment (bench) [inline src/test/java/ai/keeneye/bioformats2n5/test/N5Test.java:503]: same remark here

File: src/test/java/ai/keeneye/bioformats2n5/test/ZarrTest.java [hunk 3]
```diff
@@ -33,7 +32,11 @@
 import loci.common.services.ServiceFactory;
 import loci.formats.FormatTools;
 import loci.formats.in.FakeReader;
-import loci.formats.ome.OMEXMLMetadata;
+
+// imports for originalMetaData tests
```
Comment (bench) [inline src/test/java/ai/keeneye/bioformats2n5/test/ZarrTest.java:36]: same :)

File: src/test/java/ai/keeneye/bioformats2n5/test/N5Test.java (hunk 1 above)
Comment (ian-keeneye) [reply src/test/java/ai/keeneye/bioformats2n5/test/N5Test.java:32]: @bench Ooops good catch !! I should restore the originalMetadata tests as well.
Sadly just merged this one I'll have to do a 2nd PR for that I think.

**Pull request merged**